test/sql/deparse.sql

```diff
@@ -0,0 +1,29 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- We compare information(\d+) about manually created tabels with the ones that were recreated using deparse_table command.
+-- There should be no diff.
+
+\set ECHO errors
+
+SELECT format('%s/results/deparse_create.out', :'TEST_OUTPUT_DIR') AS "CREATE_OUT",
+       format('%s/results/deparse_recreate.out', :'TEST_OUTPUT_DIR') AS "RECREATE_OUT"
+\gset
+SELECT format('\! diff %s %s', :'CREATE_OUT', :'RECREATE_OUT') as "DIFF_CMD"
```

Comment:
Do we need to rely on an external tool here? Isn't there a way to read a file into a text variable (e.g., `\lo_import` in `psql`) and then compare them in PostgreSQL? Although I guess `pg_regress` might itself rely on `diff`?

Reply:
Well we've been using `diff` in our tests already. I find `diff` to be more standard way of comparing files that everyone is familiar with. BTW `pg_regress` does use `diff`, so I guess it's fine.